app/models/user.rb

```diff
@@ -73,6 +74,10 @@ def self.saml_update_resource_hook(user, saml_response, auth_value)
   #   super
   # end
 
+  def is_admin?
```

Comment:
rename this, has_global_assignment?, change logic to use database query rather than getting all assignments, could limit to 1 record as if they have even one global assignment this should return true.